src/App.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="app">
+    <div class="main">
+      <HeaderComponent />
+      <div class="container">
+        <CardComponent :numbersArray="$state.array" />
```

Comment:
This is not Card component as you are passing an array to it. It is not single thing